draft-ietf-cdni-uri-signing.xml

```diff
@@ -574,18 +574,23 @@
         </section>
         <section anchor="uri_container_forms" title="URI Container Forms">
           <t>The URI Container (cdniuc) claim takes one of the following forms. More forms may be added in the future to extend the capabilities.</t>
-          <t>Before comparing a URI against this container, the signed JWT must be removed from
-          the URI. This removal is only for the purpose of determining if the URI matches; all
-          other purposes will use the original URI. If the signed JWT is terminated by anything
-          other than a sub-delimiter (as definined in <xref target="RFC3986"/> Section 2.2),
-          everything from the reserved character (as defined in <xref target="RFC3986"/> Section 2.2)
-          that precedes the URI Signing Package Attribute to the last character of the signed
-          JWT will be removed, inclusive. Otherwise, everything from the first character of the
-          URI Signing Package Attribute to the sub-delimiter that terminates the signed
-          JWT will be removed, inclusive.</t>
+          <t>Before comparing a URI against this container, the following steps MUST be performed:
+          <list style="symbols">
+            <t>Remove the signed JWT from
+              the URI. This removal is only for the purpose of determining if the URI matches; all
+              other purposes will use the original URI. If the signed JWT is terminated by anything
+              other than a sub-delimiter (as definined in <xref target="RFC3986"/> Section 2.2),
+              everything from the reserved character (as defined in <xref target="RFC3986"/> Section 2.2)
+              that precedes the URI Signing Package Attribute to the last character of the signed
+              JWT will be removed, inclusive. Otherwise, everything from the first character of the
+              URI Signing Package Attribute to the sub-delimiter that terminates the signed
+              JWT will be removed, inclusive.</t>
+            <t>Normalize the URI according to <xref target="RFC3986"/> sections 6.2.2 and 6.2.3.</t>
+          </list></t>
 
           <section anchor="uri_container_forms_hash" title="URI Hash Container (hash:)">
-              <t>Prefixed with 'hash:', this string is a URL Segment form (<xref target="RFC6920"/> Section 5) of the URI.</t>
+              <t>Prefixed with 'hash:', this string is a URL Segment form (<xref target="RFC6920"/> Section 5) of the URI. Before generating the hash, the URI MUST
```

Comment:
I believe the same is true here as it is above (or point to the above in some reasonable way instead of saying it twice).